app/Jobs/FetchWalletNfts.php

```diff
@@ -65,6 +73,15 @@ public function handle(): void
         GalleryCache::clearAllDirty();
 
         UserCache::clearAll($this->wallet->user);
+
```

Comment:
a few lines above there is `$nftHandler->cleanupNftsAndGalleries($this->startTimestamp);`, maybe also add logs in there when a gallery is adjusted? that would highlight any changes that are being made to them from jobs